res/controllers/Vestax VCI-300.midi.xml

```diff
@@ -4,7 +4,7 @@
 		<name>Vestax VCI-300</name>
 		<author>Uwe Klotz</author>
 		<forums>http://www.mixxx.org/forums/viewtopic.php?f=7&amp;t=4021</forums>
-		<wiki>http://mixxx.org/wiki/doku.php/vestax_vci-300</wiki>
+        <manual>/hardware/controllers/vestax_vci_300.html</manual>
```

Comment:
Did you consider to remove the hardware/controllers part here?
This allows to only change a single place if this ever changes

Reply:
Good point, I could also remove the file extension in case we ever switch to the HTML-folders sphinx builder.

Reply:
Done.